### PR TITLE
EICNET-1098: Create a group feature for "Files"

### PR DESCRIPTION
**Pull request by 4hardfire**
### Features/Improvements

- Create group permissions: "access wiki overview" and "access files overview";
- Create group feature plugin to show/hide group menu link for the files overview page (this should take into account permission "access files overview");
- Enable or disable group feature "Wiki" based on "access wiki overview" permission; Allow public roles to view Wiki overview page of a public group.

### Tests

- [x] Create a new group and publish it
- [x] When viewing the group page, the group menu link "Files" should be enabled by default
- [x] As anonymous user or logged in user I can see the group menu link "Files" because it's a public group
- [x] As GO disable the group feature "Files"
- [x] As anonymous or logged in user, make sure you cannot see the group menu link "Files" anymore
- [x] Test the enable and disable the group feature "Wiki" and check if it has the same behavior. Note that this group feature will never be enabled by default.

### CODE REVIEW

File: lib/modules/eic_groups/eic_groups.routing.yml [hunk 1]
```diff
@@ -38,3 +38,12 @@ eic_groups.group.highlight_content:
     _highlight_group_content: 'TRUE'
     node: \d+
     group: \d+
+
+eic_overviews.groups.overview_page.files:
+  path: '/group/{group}/library'
+  defaults:
+    # @todo: update controller namespace once we have the class.
```
Comment (didierdemaeyer) [inline lib/modules/eic_groups/eic_groups.routing.yml:45]: @4hardfire Can this todo already be handled or should this be done in a different PR?

Comment (4hardfire) [reply lib/modules/eic_groups/eic_groups.routing.yml:45]: It should be handled after PR #387 gets merged.

**Review by didierdemaeyer — APPROVED** [review]
